recvalsite/settings.py

```diff
@@ -33,7 +33,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DEBUG", default=False, cast=bool)
-DEBUG_DJANGO = ""
+USE_DJANGO_DEBUG_TOOLBAR = config("DEBUG_DJANGO", default=DEBUG, cast=bool)
```

Comment:
You’ll want `USE_DJANGO_DEBUG_TOOLBAR` on the right side too, and in the `ci_run_cypress` script.